spamhandling.py

```diff
@@ -103,8 +101,7 @@ def handle_spam(title, body, poster, site, post_url, poster_url, post_id, reason
             username = poster.strip()
             user_link = poster_url
 
-        t_metasmoke = Thread(target=Metasmoke.send_stats_on_post,
```

Comment:
Leave it as it was - we broke it since the arguments are pretty long so it makes it easier to read.